tools/topology/sof/pipe-codec-adapter-playback.m4

```diff
@@ -115,10 +115,10 @@ W_CODEC_ADAPTER(0, PIPELINE_FORMAT, DAI_PERIODS, DAI_PERIODS, CA_SCHEDULE_CORE,
         LIST(`          ', "CA_SETUP_CONTROLBYTES_NAME_PIPE", "CA_RUNTIME_CONTROLBYTES_NAME_PIPE"))
 
 # Playback Buffers
-W_BUFFER(0, COMP_BUFFER_SIZE(DAI_PERIODS,
+W_BUFFER(0, COMP_BUFFER_SIZE(8,
```

Comment:
We need to figure out what's with this hardcoding.

Reply:
Yes, yes! Will fix it asap.